.ci/jobs/observability-docs-mbp.yml

```diff
@@ -0,0 +1,43 @@
+---
+- job:
+    name: apm-server/observability-docs-mbp
+    display-name: Observability Docs
+    description: Observability Docs
+    project-type: multibranch
+    concurrent: true
+    script-path: .ci/Jenkinsfile
+    scm:
+    - github:
+        branch-discovery: no-pr
+        discover-pr-forks-strategy: merge-current
+        discover-pr-forks-trust: permission
+        discover-pr-origin: merge-current
+        discover-tags: true
+        notification-context: 'apm-ci'
+        repo: observability-docs
+        repo-owner: elastic
+        credentials-id: 2a9602aa-ab9f-4e52-baf3-b71ca88469c7-UserAndToken
+        ssh-checkout:
+          credentials: f6c7695a-671e-4f4f-a331-acdce44ff9ba
+        build-strategies:
+        - tags:
+            ignore-tags-older-than: -1
+            ignore-tags-newer-than: -1
+        - regular-branches: true
+        - change-request:
+            ignore-target-only-changes: false
```

Comment:
To avoid th erebuild when the target branch has changed, we don't use that feature though since it will trigger a bunch of builds when the master branch for isntance got a new commit

```suggestion
            ignore-target-only-changes: true
```